python/paddle/fluid/tests/unittests/ir/inference/CMakeLists.txt

```diff
@@ -133,5 +134,7 @@ if (WITH_MKLDNN)
   set_tests_properties(test_mkldnn_matmul_v2_transpose_reshape_fuse_pass PROPERTIES TIMEOUT 100)
   set_tests_properties(test_mkldnn_conv_transpose_bias_fuse_pass PROPERTIES TIMEOUT 100)
   set_tests_properties(test_conv_eltwiseadd_bn_fuse_pass PROPERTIES TIMEOUT 300)
+  set_tests_properties(test_mkldnn_conv_mish_fuse_pass PROPERTIES TIMEOUT 300)
```

Comment:
If the execution time of a UT does not exceed 300 seconds, there is no need to modify the single test timeout setting.

Reply:
Locally it works with or without explicitly setting timeout. Although not setting it caused CI fail on _test_mkldnn_conv_mish_fuse_pass_